Helper/ClientManager.php

```diff
@@ -54,26 +50,38 @@ public function __construct(
         $this->moduleList = $moduleList;
     }
 
-    public function initClient($storeId = null)
+    public function getClient($storeId = null)
     {
-        $apiKey = $this->getSettingsVal(self::XML_API_KEY, $storeId);
-        $apiSecret = $this->getSettingsVal(self::XML_API_SECRET, $storeId);
-        $this->client = new MageClient($apiKey, $apiSecret, $this->getSetupVersion(), $this->logger, $this->scopeResolver);
+        $cacheKey = (int)$storeId;
```

Comment:
Are there any concerns with `$storeId` potentially being `null` and `$cacheKey` being zero?

Reply:
There are no concerns, it was intentionally made like that.